modules/nw-sriov-hwol-creating-sriov-policy.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-hardware-offloading.adoc
+
+:_content-type: PROCEDURE
+[id="configure-sriov-node-policy_{context}"]
+= Configuring the SR-IOV network node policy
+
+You can create an SR-IOV network device configuration for a node by creating an SR-IOV network node policy.
+To enable hardware offloading, you must define the `.spec.eSwitchMode` field with the value `"switchdev"`.
+
+The following procedure creates an SR-IOV interface for a network interface controller with hardware offloading.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a file, such as `sriov-node-policy.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-node-policy <.>
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice <.>
+  eSwitchMode: "switchdev" <.>
+  nicSelector:
+    deviceID: "1019"
+    rootDevices:
+    - 0000:d8:00.0
+    vendor: "15b3"
+    pfNames:
```

Comment:
deviceID should be `1017`
rootDevices should be changed according the real worker, So I think we need add a reference
pfNames: should be also changed according to real worker.

Reply:
@zshi-redhat Can you comment on this? The YAML in both policy docs you provided use `deviceID: 1019`, and this was tested. Did I miss something?

Reply:
it's ok for me with 1019.